plugin/src/main/java/com/microsoft/java/bs/gradle/plugin/JavaLanguageModelBuilder.java

```diff
@@ -67,8 +66,10 @@ public DefaultJavaExtension getExtensionFor(Project project, SourceSet sourceSet
 
       List<String> compilerArgs = getCompilerArgs(javaCompile);
       extension.setCompilerArgs(compilerArgs);
-      extension.setSourceCompatibility(getSourceCompatibility(compilerArgs));
```

Comment:
What about `getSourceCompatibility()` && `getTargetCompatibility()`. Will you consider to remove it as well? Since I believe they are not used now.

Reply:
@Arthurm1, are you feel ok to remove these unused methods? If yes, I can append a commit to the PR and merge it.

Reply:
Which ones do you mean?  `JvmBuildTargetEx.getTargetCompatibility()`?  Is that no longer used?

Or do you mean `JavaLanguageModelBuilder.getTargetCompatibility(List<String> compilerArgs)` should be removed?  That's a copy of what Gradle does [here](https://github.com/gradle/gradle/blob/dbd2232425b245f077cacb58b9c5acbc1690511e/platforms/jvm/language-java/src/main/java/org/gradle/api/internal/tasks/compile/JavaCompilerArgumentsBuilder.java#L143).  It looks like we should probably add more option handling from the Gradle code.

Reply:
Sorry, my bad. I was not aware that they are used when parsing the compiler args.